Remove polling of APML command

abinayaddhandapani opened the pull request:
Remove polling of APML command to check if the APML is up as it is breaking the processor redfish URI without no response until APML is up.
 Prevent the cpu-info service from starting automatically at boot and it will started by the set-apml.sh script once APML is up.

Commit:
Remove polling of APML command

Remove polling of APML command to check if the
APML is up as it is breaking the processor redfish URI
without no response until APML is up.

Signed-off-by: Abinaya Dhandapani <abinaya.dhandapani@amd.com>

modolaty reviewed (commented):
This change looks good to me.
Please disable cpu-info service and let set-apml.sh start and stop it

Commit:
Prevent cpu-info service from starting automatically

Prevent the cpu-info service from starting automatically at boot
and it will started by the set-apml.sh script once APML is up.

Signed-off-by: Abinaya Dhandapani <abinaya.dhandapani@amd.com>

abinayaddhandapani commented:
@modolaty  Disabled the service to start automatically during boot

abinayaddhandapani commented:
Closing this PR as disabling the service needs to be done from bb file.
